src/Lumi/Components/Form.purs

```diff
@@ -352,6 +355,25 @@ passwordBox
        String
 passwordBox = inputBox Input.password
 
+-- | A configurable textarea box makes a `FormBuilder` for strings
+-- | includes a configurable placeholder
+textarea_
+  :: forall props
+   . Textarea.TextareaProps
+  -> FormBuilder
+       { readonly :: Boolean | props }
+       String
+       String
+textarea_ textareaProps = formBuilder_ \{ readonly } s onChange ->
+  if readonly
+    then Input.alignToInput $ R.text s
+    else Textarea.textarea textareaProps
+           { value = s
+           , placeholder = textareaProps.placeholder
```

Comment:
I think this line is unnecessary, because it's setting `placeholder` to the value it already has.

Reply:
Oh yep, see what you mean. Thanks